libcontainer/configs/namespaces_linux.go

```diff
@@ -6,6 +6,7 @@ import (
 	"sync"
 )
 
+// NamespaceType constants
```

Comment:
nit: missing period.

Reply:
I am finally addressing this in https://github.com/opencontainers/runc/pull/3304, yay!